Brisk/Controllers/RadarViewController.swift

```diff
@@ -60,6 +61,9 @@ final class RadarViewController: ViewController {
         self.classificationPopUp.setItems(titles: Classification.All.map { $0.name })
         self.reproducibilityPopUp.setItems(titles: Reproducibility.All.map { $0.name })
         self.productPopUp.set(items: Product.All, getTitle: { $0.name }, getGroup: { $0.category })
+        self.attachmentDroppableView.droppedAttachment = { [weak self] attachments in
+            self?.attachments = [attachments]
```

Comment:
Should this be overridden or appended? I get we don't really support multiple attachments now, but if there's UI for this in the future I think adding makes more sense here than replacing?